react-relay/index.d.ts

```diff
@@ -128,7 +146,13 @@ declare module "react-relay" {
     }
 
     interface RelayProp {
-        variables: any
-        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void
+        route: { name: string; }; // incomplete, also has params and queries
+        variables: any;
+        pendingVariables?: any;
+        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
+        forceFetch(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
```

Comment:
I don't know what the DefinitelyTyped policy is on TS version compatibility. `variables` should also be set as `readonly` as well, but that ties this to TS 2+